test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" {
+        Get-Verb | Should not be $null
+    }
+
+    It "Should get a specific verb" {
+        (Get-Verb -Verb Add | Measure-Object).Count | Should be 1
```

Comment:
if you're not sure whether you'll get a singleton or a collection you can always add `@()`